google/resource_google_project.go

```diff
@@ -210,6 +210,17 @@ func resourceGoogleProjectCheckPreRequisites(config *Config, d *schema.ResourceD
 	if !stringInSlice(resp.Permissions, perm) {
 		return fmt.Errorf("missing permission on %q: %v", ba, perm)
 	}
+	errExpected := "googleapi: Error 403: Project '00000000000' not found or permission denied., forbidden"
```

Comment:
Does using a dummy project work here? I would expect us to need the quota project- the seed project in most cases- instead.

Reply:
Actually, this is just to make the string easier to match. The idea was discussed in issue #6222 and the suggestion was that we should use 0000000 because we always get not found, meaning the API is enabled. The API I'm using does not matter actually. I just want to force an not found.

Reply:
just friendly FYI, I've compiled and tested. It does work as expected.

Reply:
Ah, right, that makes sense because we're sending a request that's based on the quota project anyways. Can you comment inline with that context?

Reply:
```We are querying a dummy project since the call is already coming from the quota project. So, we always need to match the two types of message.``` how does it look like?

Reply:
sgtm!